core/src/main/java/jenkins/BreadCrumbItem.java

```diff
@@ -0,0 +1,43 @@
+package jenkins;
+
+
+
+public class BreadCrumbItem {
+
+    private String displayName;
+    private boolean hasContextualMenu;
+    private BreadCrumb breadCrumb;
+
+
+    public BreadCrumbItem getFirst() {
+        return breadCrumb.generateBreadCrumbs().get(0);
```

Comment:
Assuming the breadCrumbList would at no time be null since it would at least hold the root breadcrumb ie Home.